client/settings/test-mode-settings/style.scss

```diff
@@ -2,4 +2,8 @@
 	> :first-child {
 		margin-top: 0;
 	}
+
+	h4 {
```

Comment:
I included this to cleanup the extra margin showing at the top of the Test Mode card:

![Screen Shot 2021-06-02 at 4 53 44 PM](https://user-images.githubusercontent.com/6723003/120764876-e7932800-c518-11eb-981d-f1a532e03e5e.png)